recipes/icepyx/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "icepyx" %}
+{% set version = "0.3.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/icepyx-{{ version }}.tar.gz
+  sha256: 6a3ed302c4bdf10502c8e165549ae58d32c247c6bce066a23fa9547c1337ba13
+  
+build:
+  number: 0
+  noarch: python
+  script: 
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [unix]
+    - set SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [win]
```

Comment:
If it is noarch you don't need this. It will failed on the CIs here but there will be no Windows CI in the feedstock. It won't hurt either, but it will never be used.

Reply:
Okay. So we did do that but the windows CI failed. What you are saying is that would have been fine?

Reply:
Yep. What @weiji14 said in https://github.com/conda-forge/staged-recipes/pull/13866#issuecomment-797238384 is correct.

There are rare cases of things that pretend to be noarch, passes on Linux, and they we find out that should've been tested on Windows. But if you ever find on of those cases the Windows users will let you know and you'll remove the `noarch` in the feedstock later.

Reply:
Cool. Thanks for that explanation 😄